src/utils/data/test-data.json

```diff
@@ -0,0 +1,250 @@
+{
```

Comment:
Have you deleted `animals.mock.js` yet? Or changed `demo.mock.js` to a json file?

Reply:
I haven't deleted `animals.mock.js` yet as I saw that it is still referenced and used in `data-source.js` and wanted to double check with you first - I will delete it and replace the reference in `data-source.js`. 

In terms of `demo.mock.js`, I have kept it as is as this file is only reference within our tests and are not used for BE tests - i can change it to a json file as well if that's preferred?